recipes/djangocms-admin-style/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.2.7" %}
```

Comment:
Will update to latest once the feedstock is created.  Need that version on the channel for... Reasons...